### PR TITLE
added a  tooltip for fingertip duration for off-chain groups

### LINKED ISSUE #304: Add a tooltip to the fingerprint duration parameter in Dashboard
Add a tooltip to the fingerprint duration input in the Dashboard explaining what fingerprint duration is.

### PR DESCRIPTION
**Pull request by tanyas-codes**
Added a tooltip to display fingertip duration information, providing users with helpful insights into the duration of fingertip interactions.

## Description

change

## Related Issue

Fixes #304 

## Does this introduce a breaking change?

 No

### CODE REVIEW

**Review by 0xjei — COMMENTED** [review]
Hey @vplasencia, do you think it would be a good idea to keep the same colour scheme as the group size selection? I'm not sure about the current colors and I'm not aware about design schemes on tooltips. 

Maybe we could use pink lettering on a white background to keep it consistent using Bandada colors?

Besides that, the update from @tanyas-codes looks good to me and thank you so much @tanyas-codes <3

TL;DR We can merge it as-is and iterate back later on colors.

**Comment by vplasencia** [comment]
Hey @0xjei 

I get your point. We can ask the design team for feedback and change it in the future if necessary.

**Review by vplasencia — APPROVED** [review]
Thank you very much @tanyas-codes for the great work ✨

**Review by 0xjei — APPROVED** [review]

**Comment by 0xjei** [comment]
> Hey @0xjei
> 
> I get your point. We can ask the design team for feedback and change it in the future if necessary.

Yes and thank you! Let's merge it :)

**Comment by tanyas-codes** [comment]
@vplasencia @0xjei Thankyou!! Will be waiting for the future updates regarding color changes.
